src/dao/ERC20DAO.js

```diff
@@ -26,9 +26,6 @@ export default class ERC20DAO extends AbstractTokenDAO {
   }
 
   getSymbol () {
-    if (!this._symbol) {
-      throw new Error('symbol is undefined')
```

Comment:
Need to discuss
After user add FakeToken even if the icon is specified we have exception on any symbol() call.